tasks/install.yml

```
@@ -7,8 +7,9 @@
 
 - name: Install required packages (apt)
   apt:
-    name: "{{ item }}"
-  with_items:
+    name: "{{ packages }}"
+  vars:
+    packages:
```

You can directly specify the list for the apt module, it is not necessary to use a variable for that:
```suggestion
- apt:
    name:
      - slapd
      - ldap-utils
      - schema2ldif
```

---

Thanks! fixed for both apt and pacman